client/components/Users/UserFormField.jsx

```diff
@@ -61,6 +61,15 @@ export default class UserFormField extends Component {
         if (validate) additionalOptions.validate = validate;
         return (this.getFieldComponent(field, Select, additionalOptions));
       }
+      case 'InputVirtualizedSelect': {
+        const additionalOptions = {
+            options: field.options,
+            multi: field.multi,
+            displayLabelOnly: field.displayLabelOnly
+        };
+        if (validate) additionalOptions.validate = validate;
+            return (this.getFieldComponent(field, VirtualizedSelect, additionalOptions));
```

Comment:
I think the indentation here should be removed as follows:

```suggestion
        return (this.getFieldComponent(field, VirtualizedSelect, additionalOptions));
```